src/lint/lint.ts

```diff
@@ -106,6 +142,9 @@ export async function validateFile(
   filename: string,
   profile: string
 ): Promise<client.validate.ValidationReport> {
+  // Initialize AMF so that we have a clean environment to work with
+  await AMF.init();
```

Comment:
The lint command calls this function and the tests call this directly so it makes sense that we initialize AMF here. With this part being in the command we'll have to do it in the tests every time we add any tests that depend on it.

Reply:
Moving it here means that we init for every single file we parse, rather than just once before all of them.
```ts
before(async () => AMF.init());
```
That's all you need in a test file. It's also what's already done in the existing tests.

Reply:
I keep forgetting that we support providing multiple files with the lint command now. I know it's just adding a line and it's already present in the other files. The issue is that if someone who isn't very familiar with the code forgets to add this, they won't realize easily that this is what they are missing.

Reply:
And it makes sense to have a clean slate for every validate operation. Since we are only doing it once for every raml it I think it should be alright.